setup.py

```diff
@@ -52,6 +52,7 @@ def _get_dependencies(requirements_file: Path) -> List[str]:
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python :: 3',
     ],
+    python_requires='>=3.0',
```

Comment:
The oldest working version (judging by the test runs I made) is 3.6.1, so:

```suggestion
    python_requires=">=3.6.1",
```